hadoop-hdds/docs/content/security/SecurityWithRanger.md

```diff
@@ -46,6 +46,12 @@ Property|Value
 ozone.acl.enabled         | true
 ozone.acl.authorizer.class| org.apache.ranger.authorization.ozone.authorizer.RangerOzoneAuthorizer
 
+To use the RangerOzoneAuthorizer, you also need to add the following environment variables to ozone-env.sh:
+```
+export OZONE_CLASSPATH="${OZONE_HOME}/share/ozone/lib/*:${OZONE_HOME}/share/ozone/lib/libext/*:${OZONE_HOME}/share/ozone/lib/libext/ranger-ozone-plugin-impl/*"
```

Comment:
Items defined in `OZONE_CLASSPATH` are added to the regular classpath, which already includes items from `share/ozone/lib` as necessary for each Ozone component.  So `share/ozone/lib/*` should not be added to `OZONE_CLASSPATH`.

Reply:
You're right. Don't have to add `share/ozone/lib/*` to `OZONE_CLASSPATH`.